src/Endpoints/MediaFile.php

```diff
@@ -14,11 +14,6 @@ class MediaFile extends Endpoint
     /**
      * Retrieve subtitles for an item.
      *
-     * @param string $video_id
-     *   The item's record ID.
```

Comment:
This seems like it could be valuable context to know that the video ID is a Record ID.

Reply:
Could I rename the variable and achieve clarity that way?

Reply:
See below.